### ISSUE TITLE
Multiple processing

### ISSUE THREAD

**Issue by KhramtsovMaksim** [nchutchind/vue-outside-events#5]
Hi.
Firstly would say what it is awesome directive and very useful!
Thank you for contribution into vue community.

But could you please slightly improve this one.

1. Could you please process several directives on one node? (unbinding work incorrectly)

2. Could you please change processing "CustomEventOutside" directive with several event names.

Thank you!

**Comment by nchutchind** [comment]
Sorry for the delay. Github didn't make me aware that there was activity on this repo.

1. What do you mean by several directives? Why would you need more than one directive?

2. What do you have in mind for event names? The custom event uses a generic name for the directive because the name has to be specified at install (as far as I'm aware).

**Comment by anlek** [comment]
Hey @nchutchind,
I think the issue is if you have `v-click-outside="doStuff"` and `v-focus-outside="doStuff"`. When the Vue UI is removed, the listeners don't seem to unregister correctly. Seems that even though the items with the directives are no longer on the page, when I click "outside" of the item (as they're no longer on the page) the function `doStuff` still gets called.

**Comment by anlek** [comment]
I think the issue is that the first `unbind` method removes the `el.__vueEventOutside__` value and the second `unbind` call no longer has any `__vueEventOutside__` value.

From `vue-outside-events/src/vue-outside-events.js` line 41
```
  outsideEvent.unbind = function (el, binding) {
    document.removeEventListener(eventName, el.__vueEventOutside__)
    el.__vueEventOutside__ = null
  }
```

I feel that you might be better off storing the `handler` inside the `outsideEvent` object.

**Comment by nchutchind** [comment]
This should be fixed in version 1.1.3 now. Let me know if you see any problems.

I just changed it to use el[`__vueEventOutside__${eventName}`] instead of just the static string. This means, of course, that you can't have multiple v-click-outside (or whatever) events on an element, but that shouldn't be an issue anyway since a single handler is all that's needed for each event on an element.

**Comment by anlek** [comment]
It seems like it works! Thank you so much for the quick fix!
